DOCS-2455: Add Mastercard and Visa requirements to Magento Vault (#1793)
MultiSafepay/docs: content/payments/integrations/ecommerce-platforms/magento2/faq/activating-magento-vault.md

    - /payments/integrations/ecommerce-platforms/magento2/faq/activating-magento-vault/
---

> Our plugin supports Magento Vault, which lets customers securely store their payment details as a token in the Magento database. This makes subsequent payments faster and easier, increasing conversion. At checkout, customers can simply select the **Instant purchase** button and don't have to re-provide their CVC. Magento Vault uses [card-on-file tokenziation](/features/recurring-payments/).

Author: ```suggestion
Our plugin supports Magento Vault, which lets customers securely store their payment details as a [card-on-file token](/features/recurring-payments] in the Magento database. This makes subsequent payments faster and easier, increasing conversion. At checkout, customers can simply select the **Instant purchase** button and don't have to re-provide their CVC. 
```

    - /payments/integrations/ecommerce-platforms/magento2/faq/activating-magento-vault/
---

> Our plugin supports Magento Vault, which is essentially a way of processing [recurring payments](/features/recurring-payments/) via the Magento payment gateway using card on file tokenziation. 

Author: ```suggestion
Our plugin supports Magento Vault, which lets customers securely store their payment details as a token in the Magento database. This makes subsequent payments faster and easier, increasing conversion. At checkout, customers can simply select the **Instant purchase** button and don't have to re-provide their CVC. Magento Vault uses [card-on-file tokenziation](/features/recurring-payments/).
```

Magento Vault:

> - Stores customers' payment details as a token in the Magento database

Author: ```suggestion
- Stores the customer's payment details as a token in the Magento database
```

> Magento Vault:

Author: ```suggestion
```

> - Stores the customer's payment details as a token in the Magento database
> - Facilitates a fast, secure checkout process
> - Increases [conversion](/getting-started/glossary/#conversion-rate)

Author: ```suggestion
```
